packages/breadcrumb/src/Breadcrumb.styles.ts

```diff
@@ -0,0 +1,9 @@
+import { styled } from '@project44-manifest/react-styles';
+
+export const StyledBreadcrumb = styled('ol', {
+  d: 'flex',
+  listStyle: 'none',
+  margin: 0,
+  p: 0,
```

Comment:
Do we have anty convention around opinionated spacing within Manifest, or is deferring to consumer the right move here?

Reply:
Does resetting user agent styles (for `ol`) fall under the category of necessary for our component? That's what the margin and padding reset is doing here. I feel like yes

Reply:
yes I think the component should provides any resets if possible